packages/backend/src/api/routers/StatusRouter.ts

```diff
@@ -1,8 +1,8 @@
 import Router from '@koa/router'
+import { stringAs } from '@l2beat/shared'
```

Comment:
why not `shared-pure`?

Reply:
good question, I'll move it there since it fits the bill of that package